packages/react-aria-components/src/Calendar.tsx

```diff
@@ -315,10 +315,15 @@ export interface CalendarCellRenderProps {
    * Whether the cell is part of an invalid selection.
    * @selector [data-invalid]
    */
-  isInvalid: boolean
+  isInvalid: boolean,
+  /**
+   * Whether the cell is today.
+   * @selector [data-today]
+   */
+  isToday: boolean
 }
 
-export interface CalendarGridProps extends StyleProps {
+export interface CalendarGridProps extends StyleProps, Omit<AllHTMLAttributes<HTMLTableElement>, 'children'> {
```

Comment:
🤔

Reply:
Needed for `cellPadding`, I could `Pick` instead, but it seemed more useful to allow all the table properties

Reply:
I think that is adding a lot more properties than just table-related ones (it's all html attributes). So Pick would definitely be better...

cellPadding and cellSpacing are technically deprecated in html too though. Do you think people ever want the default padding? Could we get away with setting it to zero by default or would that break things?

Reply:
you mean as opposed to doing it via css selectors like `.calendar td`?
I do not know.
Would we default it to 0 via the deprecated attribute? or inline style? or....?

Reply:
you mean as opposed to doing it via css selectors like `.calendar td`?
I do not know.
Would we default it to 0 via the deprecated attribute? or inline style? or....?